task/apply-tags/0.1/README.md

```diff
@@ -0,0 +1,13 @@
+# apply-tags task
+
+Apply-tags task will apply additional tags to the specified IMAGE. These additional tags can be provided via the ADDITIONAL_TAGS array parameter or they can also be provided in the image label "konflux.additional-tags". If you specify more than one additional tag in the label, they must be separated by a comma, e.g:
```

Comment:
> they must be separated by a comma, e.g:

Technically, it also works if they are space-separated 

```bash
$ IFS=', ' read -r -a array <<< 'v1 v1.0 latest'
$ printf "%s\n" "${array[@]}"
v1
v1.0
latest
```

That's probably a good thing

Reply:
Good point. I updated the README docs instead of changing the behavior